sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -74,6 +74,61 @@ export interface DeleteConfigurationSettingOptions extends HttpOnlyIfUnchangedFi
 export interface DeleteConfigurationSettingResponse extends SyncTokenHeaderField, HttpResponseFields, HttpResponseField<SyncTokenHeaderField> {
 }
 
+// @public
+export interface FeatureFlag extends FeatureFlagParam, ConfigurationSetting {
+}
+
+// @public
+export const featureFlagContentType = "application/vnd.microsoft.appconfig.ff+json;charset=utf-8";
```

Comment:
Do we expect customers to actually use this? Wondering if we can mark it as `@internal`.

Reply:
This is the only thing that differentiates a feature flag config setting from a regular config setting.
We need users to provide this string in the setting as content-type so that it can be serialized properly while sending the request.